modules/layers/src/text-layer/multi-icon-layer/multi-icon-layer.ts

```diff
@@ -102,18 +103,30 @@ export default class MultiIconLayer<DataT, ExtraPropsT = {}> extends IconLayer<
   draw(params) {
     const {sdf, smoothing, outlineWidth} = this.props;
     const {outlineColor} = this.state;
+    const outlineBuffer = outlineWidth ? DEFAULT_BUFFER * (1 - outlineWidth) : -1;
```

Comment:
As discussed, this value needs an upper limit to avoid visual artifacts.

Reply:
I've reverted to using the original method of calculating `outlineBuffer` and it appears to work well, i.e. no artefacts and large `outlineWidth` values result in an outline clamped at the upper limit.